### ISSUE TITLE
yarrrml-parser breaks with Node v16

### ISSUE THREAD

**Issue by julianrojas87** [RMLio/yarrrml-parser#149]
### Description
YARRRML parsing fails when running under Node v16.  It looks like an issue of the graphy.js dependency according to the logs:
```bash
$ yarrrml-parser -i ../test/mappings/ERATV-vehicle-types.yml 
/home/julian/Desktop/yarrrml-parser/node_modules/@graphy/memory.dataset.fast/main.js:776
				si_norm_gs += sc1_subject+'\8';
				                           ^

SyntaxError: \8 and \9 are not allowed in strict mode.
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/julian/Desktop/yarrrml-parser/lib/tools.js:19:17)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
```
With Node version <=14 seems to work fine. 

### Environment
- Node v16.13.0
- Ubuntu v20.04

**Comment by DylanVanAssche** [comment]
I can reproduce this issue.
Seems like an issue with some NodeJS library, question is which of course as this is not a direct dependency.

**Comment by bjdmeest** [comment]
Ah yes, graphy broke in v4.3.4 (see https://github.com/blake-regalia/graphy.js/issues/49 ), sticking the version to v4.3.3 fixes it

**Comment by julianrojas87** [comment]
I see, then it is needed to fix graphy's version 4.3.3 [here](https://github.com/RMLio/yarrrml-parser/blob/f33f6fc8e2420a713dc4a13d5e708b41439ba8a6/package.json#L25) until its problems are fixed or drop it completely and rely on e.g. n3.js entirely (I guess this might be more difficult).

Happy to provide a PR with the easy fix ;)

**Comment by DylanVanAssche** [comment]
@julianrojas87 Sure, feel free to make a PR :)

**Comment by DylanVanAssche** [comment]
I have a fix incoming, stay tuned!

**Comment by DylanVanAssche** [comment]
@julianrojas87 Can you try the latest `development` branch to see if your issue is fixed?

**Comment by DylanVanAssche** [comment]
Fixed in v1.3.4
